src/Esfa.Vacancy.Register.Api/Views/Shared/_Layout.cshtml

```diff
@@ -19,12 +19,6 @@
     <link href="~/Content/dist/css/screen.css" media="all" rel="stylesheet" />
     <!--<![endif]-->
 
-    <style>
-        #api_info {
```

Comment:
Can you explain why you removed this?

Reply:
Moved it into screen.css rather than removed as inline CSS violates Front End separation of concerns.